__tests__/helpers.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const MemoryFileSystem = require("memory-fs");
+const webpack = require("webpack");
+
+exports.PluginEnvironment = class PluginEnvironment {
```

Comment:
👍 Can we make this more generic and could you elaborate a bit more in detail about it, I would like to write docs about it then, for the `webpack-defaults` test boilerplate 

`ExpectedPlugin` `TestPlugin` `JestPlugin`?